omero/index.html

```diff
@@ -7,7 +7,16 @@
             <div class="row homepage text-center bg-image-text-container">
                 <div class="small-12 columns">
                     <h1 class="hero-main-header"><img src="{{ site.baseurl }}/img/logos/omero-white.svg" alt="OMERO logo" class="logo-hero" /></h1>
-                    <p class="hero-subheader small-12">From the microscope to publication, OMERO handles all your images in a secure central repository. You can view, organize, analyze and share your data from anywhere you have internet access. Work with your images from a desktop app (Windows, Mac or Linux), from the web or from 3rd party software. Over 140 image file formats supported, including all major microscope formats.</p>
+                    <p class="hero-subheader small-12">
+                        From the microscope to <a href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/publication.html" target="blank">publication</a>,
```

Comment:
I'm a little skeptical of having the word `publication` link out to the docs (long-term). I could see having each of the features here link to the feature section (`view`, `organize`, `analyze`, etc.) especially once https://trello.com/c/nkawELEq/134-add-sample-content-to-feature-pages is in place.

Reply:
Ah! That's a nice idea. I could have them link out to the feature sections now rather than this odd one link and then change it again.